core/src/main/java/org/apache/carbondata/core/scan/result/BlockletScannedResult.java

```diff
@@ -873,12 +872,11 @@ public DeleteDeltaVo getCurrentDeleteDeltaVo() {
   /**
    * Below method will be used to check row got deleted
    *
-   * @param rowId
    * @return is present in deleted row
    */
-  public boolean containsDeletedRow(int rowId) {
+  public boolean containsDeletedRow(int currentRow) {
```

Comment:
Is this still a row ID? Seems rowId is better

Reply:
ok